docs/src/design/stack/io_ops.md

```diff
@@ -21,22 +21,22 @@ $$
 The effect of this operation on the rest of the stack is:
 * **Right shift** starting from position $0$.
 
-### READ
-Assume $a$ is an element at the head of the advice stack. The `READ` operation removes $a$ from the advice stack and pushes it onto the stack. The diagram below illustrates this graphically.
+### ADVPOP
+Assume $a$ is an element at the head of the advice stack. The `ADVPOP` operation removes $a$ from the advice stack and pushes it onto the stack. The diagram below illustrates this graphically.
```

Comment:
Let's update the terminology here and in the other places in this file to be consistent with how we use it in other places (e.g., instead of "head of the advice stack" we should say "top of the advice stack").

Reply:
Actually, the part about imposing constraints should be about the operand stack. We don't impose any constraints against advice stack in general, but we usually have constraints against the operand stack. In this specific case, a portion of the operand stack is unconstrained.

Reply:
Fixed!